changelog/changelog2022.md

```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
+
+* [Source](../mews-operations/reservations.md#source) replaced Channel in the [Process Group Request](../mews-operations/reservations.md#request), which now supports processing of multiple sources.
+* [Travel Agency](../mews-operations/reservations.md#company) was added to [Process Group Request](../mews-operations/reservations.md#request).
```

Comment:
wrong link to travel agency

Reply:
Is it true that the meaning of Company has changed? I'm not sure myself, but I suspect in the old API, 'Company' could mean either a Travel Agent or a Corporate, now this has been split into two, so 'Company' now only ever means a Corporate, and a new property has been added 'Travel Agency' which means a Travel Agent. I hope I'm making sense!